### PR TITLE
🚧 More homepage adjustments

### PR DESCRIPTION
**Pull request by lluerich**
- adjust some more styles
- adds newsletter switch

### CODE REVIEW

File: pages/content/amp-dev/documentation/guides-and-tutorials/contribute/integrate-your-analytics-tools.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ toc: true
 ---
 
 <!--
-This file is imported from https://github.com/ampproject/amphtml/blob/main/extensions/amp-analytics/integrating-analytics.md.
+This file is imported from https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/integrating-analytics.md.
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/contribute/integrate-your-analytics-tools.md:13]: This looks like an outdated file -> should be main

Comment (lluerich) [reply pages/content/amp-dev/documentation/guides-and-tutorials/contribute/integrate-your-analytics-tools.md:13]: I will merge the latest changes again. Something went wrong there, I guess.

Comment (lluerich) [reply pages/content/amp-dev/documentation/guides-and-tutorials/contribute/integrate-your-analytics-tools.md]: This looks better now.

File: frontend21/scss/amp-dev.scss [hunk 2]
```diff
@@ -45,6 +45,7 @@
 @import 'components/content/teaser.scss';
 @import 'components/content/intro.scss';
 @import 'components/extra/about/components-showcase.scss';
+@import 'components/newsletter.scss';
```
Comment (matthiasrohmer) [inline frontend21/scss/amp-dev.scss:48]: Can you move this to a folder named `extra/home` as this is a component solely used on the homepage?

**Review by matthiasrohmer — APPROVED** [review]
Only one small request.

File: frontend21/scss/amp-dev.scss (hunk 2 above)
Comment (lluerich) [reply frontend21/scss/amp-dev.scss]: Yes, that makes total sense. Thanks.

**Pull request merged**